userguide/content/en/docs/Language/_index.md

```diff
@@ -98,10 +98,14 @@ If you configure more than one language in `config.toml`, the Docsy theme adds a
 
 ## Internationalization bundles
 
-All UI strings (text for buttons etc.) are bundled inside `/i18n` in the theme, with a `.toml` file for each language.
+All UI strings (text for buttons, repository links, etc.) are bundled inside `/i18n` in the theme, with a `.toml` file for each language.
 
-If your chosen language isn't currently in the theme and you create your own `.toml` file for all the common UI strings (for example, if you translate the UI text into Japanese and create a copy of `en.toml` called `jp.toml`), we recommend you do this **in the theme** rather than in your own project, so it can be reused by others. Any additional strings or overridden values can be added to your project's `/i18n` folder.
+If your chosen language isn't currently in the theme and you create your own `.toml` file for all the common UI strings (for example, if you translate the UI text into Japanese and create a copy of `en.toml` called `jp.toml`), we recommend you do this **in the theme** rather than in your own project, so it can be reused by others. 
```

Comment:
Remarks:
- `jp.toml`: according to [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), the two-letter code for Japan is `ja`, so I would opt for `ja.toml`.
- The docsy repo already has [`ja.toml`](https://github.com/google/docsy/blob/main/i18n/ja.toml), so I would propose to use a different language that does not exist yet.

Reply:
Good suggestion! Done.